python/ray/serve/_private/proxy_state.py

```diff
@@ -30,35 +32,174 @@
 logger = logging.getLogger(SERVE_LOGGER_NAME)
 
 
+class ActorWrapper(ABC):
+    def __init__(self, actor_handle: ActorHandle):
+        self._actor_handle = actor_handle
+        self._ready_obj_ref = None
+        self._health_check_obj_ref = None
+        self._is_drained_obj_ref = None
+        self._update_draining_obj_ref = None
+
+    @property
+    def actor_id(self) -> str:
+        return self._actor_handle._actor_id.hex()
+
+    @property
+    def actor_handle(self) -> ActorHandle:
+        return self._actor_handle
```

Comment:
This should not be a property of the wrapper, it's exposing a ray API. Why do we need to be able to get the raw actor handle?

Reply:
I think this all boiled down to the `_start_controller()` call to check proxies are running https://github.com/ray-project/ray/blob/master/python/ray/serve/_private/api.py#L186 There are also a few other tests that's depended on this. 

I can also see that maybe we just keep a instance variable for it instead of dynamically generate in https://github.com/ray-project/ray/blob/master/python/ray/serve/_private/proxy_state.py#L363 Can talk offline